autoscheduler/frontend/src/tests/ui/CourseSelectCard.test.tsx

```diff
@@ -806,438 +697,4 @@ describe('Course Select Card UI', () => {
-      store.dispatch(setTerm('201931'));
-      const {
-        getByLabelText, getByText, findByText, findAllByText,
-      } = render(
-        <Provider store={store}><CourseSelectCard id={0} /></Provider>,
-      );
-
-      // act
-      // fill in course and go to section view
-      const courseEntry = getByLabelText('Course') as HTMLInputElement;
-      fireEvent.change(courseEntry, { target: { value: 'CSCE ' } });
-      fireEvent.click(await findByText('CSCE 121'));
-
-      fireEvent.click(getByText('Section'));
-
-      // assert
-      // verify text for seats is correct
-      const sectionText = (await findAllByText(/ seats left/))[0];
-      expect(sectionText).toHaveTextContent('25/25 seats left');
-    });
-
-    test('when a section has max enrollment', async () => {
-      // arrange
-      fetchMock.mockResponseOnce(JSON.stringify({ // api/course/search
-        results: ['CSCE 121', 'CSCE 221', 'CSCE 312'],
-      }));
-      fetchMock.mockImplementationOnce(testFetch); // api/sections
-
-      const store = createStore(autoSchedulerReducer, applyMiddleware(thunk));
-      store.dispatch(setTerm('201931'));
-      const {
-        getByLabelText, getByText, findByText, findAllByText,
-      } = render(
-        <Provider store={store}><CourseSelectCard id={0} /></Provider>,
-      );
-
-      // act
-      // fill in course and go to section view
-      const courseEntry = getByLabelText('Course') as HTMLInputElement;
-      fireEvent.change(courseEntry, { target: { value: 'CSCE ' } });
-      fireEvent.click(await findByText('CSCE 121'));
-
-      fireEvent.click(getByText('Section'));
-
-      // assert
-      // verify text for seats is correct
-      const sectionText = (await findAllByText(/ seats left/))[1];
-      expect(sectionText).toHaveTextContent('0/25 seats left');
-    });
-
-    test('when a section has over max enrollement', async () => {
-      // arrange
-      fetchMock.mockResponseOnce(JSON.stringify({ // api/course/search
-        results: ['CSCE 121', 'CSCE 221', 'CSCE 312'],
-      }));
-      fetchMock.mockImplementationOnce(testFetch); // api/sections
-
-      const store = createStore(autoSchedulerReducer, applyMiddleware(thunk));
-      store.dispatch(setTerm('201931'));
-      const {
-        getByLabelText, getByText, findByText, findAllByText,
-      } = render(
-        <Provider store={store}><CourseSelectCard id={0} /></Provider>,
-      );
-
-      // act
-      // fill in course and go to section view
-      const courseEntry = getByLabelText('Course') as HTMLInputElement;
-      fireEvent.change(courseEntry, { target: { value: 'CSCE ' } });
-      fireEvent.click(await findByText('CSCE 121'));
-
-      fireEvent.click(getByText('Section'));
-
-      // assert
-      // verify text for seats is correct
-      const sectionText = (await findAllByText(/ seats left/))[2];
-      expect(sectionText).toHaveTextContent('-1/25 seats left');
-    });
-  });
```

Comment:
my god I never realized how absolutely _massive_ this file was, great call splitting this up